text/3463-crates-io-policy-update.md

```diff
@@ -0,0 +1,224 @@
+- Start Date: 2023-07-24
+- RFC PR: [rust-lang/rfcs#3463](https://github.com/rust-lang/rfcs/pull/3463)
+
+# Summary
+[summary]: #summary
+
+The Rust community has outgrown the current crates.io policies. This RFC proposes a new "Terms of Use" policy based on prior work by PyPI, npm and GitHub.
+
+# Motivation
+[motivation]: #motivation
+
+> Why are we doing this? What use cases does it support? What is the expected outcome?
+
+crates.io has a "[Package Policies](https://crates.io/policies)" page, which describes the current, organically grown policies. A lot of support requests or questionable uses of crates.io however need explicit decisions from the crates.io team, since many cases are currently not covered by these policies. Additionally, decisions made by the team may be seen as arbitrary without written guidelines.
+
+The situation around name squatting has lately also reached unsustainable levels, and while [namespaces](https://github.com/rust-lang/rfcs/pull/3243) might help with some parts of the problem we would still need policies for name squatting namespaces then.
+
+The main motivation for this RFC is to give the crates.io team a fixed set of rules to determine if a project is using crates.io in a reasonable way, or if the user should get a warning and the project potentially be removed. It is mostly codifying the existing practices of the team, except for being more strict regarding name squatting.
+
+# Proposal
+[guide-level-explanation]: #guide-level-explanation
+
+The following is a proposed new "Terms of Use" policy for crates.io, replacing <https://crates.io/policies> and <https://crates.io/data-access>.
+
+-------------------------------------------------------------------------------
+
+# Terms of Use
+
+**Short version:** _crates.io is a critical resource for the Rust ecosystem,
+which hosts a variety of packages from a diverse group of users. That resource
+is only effective when our users are able to work together as part of a
+community in good faith. While using crates.io, you must comply with our
+Acceptable Use Policies, which include some restrictions on content and conduct
+on crates.io related to user safety, intellectual property, privacy,
+authenticity, and other limitations. In short, be excellent to each other!_
+
+We do not allow content or activity on crates.io that:
+
+- violates the [Code of Conduct](https://www.rust-lang.org/policies/code-of-conduct)
+  of the Rust project
+- is unlawful or promotes unlawful activities
```

Comment:
This also should be clarified based upon jurisdiction. For example, it is unlawful to be gay in some countries, but that should not be against crates.io guidelines.

Reply:
```suggestion
- is unlawful or promotes unlawful activities incurring legal liability in the countries the Rust Foundation officially operates in
```

on Zulip the suggestion above came up, which might help? (the Rust Foundation is the official legal entity operating crates.io)

Reply:
I think that's very reasonable!

Reply:
✅